torch/testing/_internal/common_methods_invocations.py

```diff
@@ -863,6 +879,22 @@ def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
 
         return samples
 
+    def reference_inputs(self, device, dtype, requires_grad=False, **kwargs):
+        """
+        Returns an iterable of SampleInputs.
+
+        Distinct from sample_inputs() above because this returns an expanded set
+        of inputs when reference_inputs_func is defined. If undefined this returns
+        the sample inputs.
+        """
+        if self.reference_inputs_func is None:
+            return self.sample_inputs_func(self, device, dtype, requires_grad, **kwargs)
+
+        if 'include_conjugated_inputs' in kwargs and kwargs.get('include_conjugated_inputs'):
```

Comment:
```suggestion
        if kwargs.get('include_conjugated_inputs', False):
```

Reply:
Included this improvement (here an to the existing code it's copied from)